index.html

```diff
@@ -0,0 +1,100 @@
+</head>
+<body>
+
+    <header class="heading">
+        <!-- CELPHONE BOX -->
+
+        <div class="phone-box">
+            <div class="time">
+                <time datetime="9:41">9:41</time>
+            </div>
+
+            <div class="cell-info">
+                <img src="Images/CellularConnection.svg" alt="">
+                <img src="Images/Wifi.svg" alt="">
+                <img src="Images/Battery.svg" alt="">
+            </div>
+            
+        </div>
+    
+
+<!-- ########### NAVBAR ############### -->
+        <nav class="nav-header">
+            <!--         LOGO -->
+            <div class="logo">
+                My Logo
+            </div>
+            <!--         MENU -->
+            <div class="menu">
+                <!-- <ul>
+                    <li>About Me</li>
+                    <li>My Projects</li>
+                    <li>Contact Me</li>
+                </ul> -->
+
+                <svg  width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
+                </svg>
+                    
+            </div>
+            
+        </nav>
+
+        
+    </header>
+
+    <div class="main">
+        
+        <h1 class="title-name">Hey There. I'm Ulises.</h1>
+        <h2 class="title-about">I'm a Software Developer</h2>
+        
+        
+        <p class="presentation">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hesitate to contact me.
+        
+        </p>
+        
+
+        <div class="social">
+            <a href="https://github.com/ulises2607" class="soc-icon">
+                <img src="Images/Social/GitHub.svg" alt="github-profile">
+            </a>
+
+            <a href="https://www.linkedin.com/in/cesar-ulises-cappa-subelza/" class="soc-icon">
+                <img src="Images/Social/Linkedin.svg" alt="linkedin-profile">
+            </a>
+
+            <a href="https://www.angellist.com/" class="soc-icon">
+                <img src="Images/Social/Angelist.svg" alt="Angelist-mainpage">
+            </a>
+
+            <a href="www.twitter.com" class="soc-icon">
+                <img src="Images/Social/Twitter.svg" alt="twitter-mainpage">
+            </a>
+
+            <a href="https://medium.com/" class="soc-icon">
+                <img src="Images/Social/Medium.svg" alt="medium-mainpage">
+            </a>
+            
+            
+        </div>
```

Comment:
- [ ] Please use list tags i.e. `<ul >`,`<li>` tag instead of `div `tag for the list of social link icons to group them, `<li>` means an item is in a list and that lets parsers (browsers, search engines, spiders) know that you're listing items. If you use `DIV` instead of `LI` those parsers would never know that those items are being listed and `DIV` does not really describe anything except that it's a block.